test.properties

```diff
@@ -40,3 +40,4 @@ org.opentosca.container.collaboration.ports=
 org.opentosca.deployment.tests=false
 org.opentosca.bus.management.mocking=false
 org.opentosca.engine.ia.keepfiles=true
+org.opentosca.test.local.repository.path=/tmp/tosca-definitions-public
```

Comment:
Don`t we need org.opentosca.test.remote.repository.url in the test.properties too?

Reply:
Only if the test cases clone the test repository themselves which is not the case